### ISSUE TITLE
Lich King Valkyrs bugg

### ISSUE THREAD

**Issue by zyklonb** [dalaranwow/dalaran-wow#5406]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
A valkyr is heading for a warlock on the outer edges of the platform only to stop and go back to the middle and grab a random raidmember, might be some distance involved, I stepped closer to the middle and it didnt happen to me as much as the warlock that were further out.

**Expected behaviour**: Tell us what should happen instead
It should grab the warlock and fly out to drop him.

**Steps to reproduce the problem**:

1.  Be a warlock (or something else and fall to your death)
2. Be on outer edge of the platform
3. Get picked by valkyr

**Include proofs for this behaviour**
https://youtu.be/tIpe57QnPGc

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
